inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java

```diff
@@ -99,137 +96,163 @@ public static void main(String[] args) {
+
+                Mat img = processedFramesQueue.poll(waitingTime, TimeUnit.SECONDS);
                 int maxProposalCount = detection.length / 7;
 
                 for (int curProposal = 0; curProposal < maxProposalCount; curProposal++) {
                     int imageId = (int) detection[curProposal * 7];
-                    if (imageId < 0)
-                        break;
-        
+                    if (imageId < 0) break;
+
                     float confidence = detection[curProposal * 7 + 2];
 
                     // Drawing only objects with >70% probability
-                    if (confidence < CONFIDENCE_THRESHOLD)
-                        continue;
-                    
+                    if (confidence < CONFIDENCE_THRESHOLD) continue;
+
                     int label = (int) (detection[curProposal * 7 + 1]);
                     int xmin = (int) (detection[curProposal * 7 + 3] * img.cols());
                     int ymin = (int) (detection[curProposal * 7 + 4] * img.rows());
                     int xmax = (int) (detection[curProposal * 7 + 5] * img.cols());
                     int ymax = (int) (detection[curProposal * 7 + 6] * img.rows());
-                    
+
                     // Draw rectangle around detected object.
-                    Imgproc.rectangle(img, new Point(xmin, ymin), new Point(xmax, ymax), new Scalar(0, 255, 0), 2); 
+                    Imgproc.rectangle(
+                            img,
+                            new Point(xmin, ymin),
+                            new Point(xmax, ymax),
+                            new Scalar(0, 255, 0),
+                            2);
                 }
 
-                if (resultCounter == warmupNum) {    
+                if (resultCounter == warmupNum) {
                     tm.start();
                 } else if (resultCounter > warmupNum) {
                     tm.stop();
-                    double worksFps = ((double)(resultCounter - warmupNum)) / tm.getTimeSec();
-                    double readFps = ((double)(framesCounter - warmupNum)) / tm.getTimeSec();        
+                    double worksFps = ((double) (resultCounter - warmupNum)) / tm.getTimeSec();
+                    double readFps = ((double) (framesCounter - warmupNum)) / tm.getTimeSec();
                     tm.start();
 
-                    Imgproc.putText(img, "Reading fps: " + String.format("%.3f", readFps), new Point(10, 50), 0 , 0.7, new Scalar(0, 255, 0), 1);
-                    Imgproc.putText(img, "Inference fps: " + String.format("%.3f", worksFps), new Point(10, 80), 0 , 0.7, new Scalar(0, 255, 0), 1);
+                    Imgproc.putText(
+                            img,
+                            "Reading fps: " + String.format("%.3f", readFps),
+                            new Point(10, 50),
+                            0,
+                            0.7,
+                            new Scalar(0, 255, 0),
+                            1);
```

Comment:
Seems like all the arguments which take more than 80 symbols we will have this row-wise layout. Please try
```java
Imgproc.putText(img, "Inference fps: " + String.format("%.3f", worksFps),
                new Point(10, 80), 0 , 0.7, new Scalar(0, 255, 0), 1);
```

Reply:
doesn't work